includes/Story_Post_Type.php

```diff
@@ -501,7 +501,7 @@ public function filter_site_kit_gtag_opt( $gtag_opt ) {
 			return $gtag_opt;
 		}
 
-		$post = get_post();
+		$post = get_queried_object();
```

Comment:
Can you elaborate on this change?

Reply:
I changed it, as seemed cleaner. But I will change it back.